src/templates/prototypes/article.pug

```diff
@@ -24,3 +22,38 @@ block content
               li If you are asking a substantive question and have an interest to declare, tick the box and email tableoffice@parliament.uk to say what the interest is.
           li Take your question form to the Table Office or Procedural Hub in person, or ask another MP to take it for you. Or sign the question form and post it or ask your staff to take it to the Table Office or Procedural Hub (faxed, stamped or photocopied signatures cannot be accepted).
           li The Table Office will check your question and contact you if they have a problem. Once the Table Office has approved your question, it has been tabled.
+
+      aside
+        .block.theme--grey-4
```

Comment:
Would be better to extend theme--grey-4 to `aside block`

This will become the default colour, we can also override this with other themes